.bcr/metadata.template.json

```diff
@@ -0,0 +1,15 @@
+{
+  "homepage": "https://github.com/bazel-ios/rules_ios",
+  "maintainers": [
+    {
+      "email": "heyluispadron@gmail.com",
+      "github": "luispadron",
+      "name": "Luis Padron"
+    }
+  ],
```

Comment:
@jszumski @thiagohmcruz @mattrobmattrob @jerrymarino @justinseanmartin please provide the information needed here, you're the first that come to mind as active maintainers here but if we have more people we'd like listed please provide those here too.

I dont have access to the repo settings but maybe listing all users with approval / merge access here makes sense.

Reply:
Is this 100% necessary to add this field? We may have to be mindful of what emails to add to the repo - will try to start a thread offline

Reply:
Seems to be required from the template/other repos I'm seeing. We could make this that bot account though if we make it

Reply:
Yeah ideally we can setup a google group / account for it

Reply:
Would be interesting to try excluding the `maintainers` but just guessing that's a possibility based on https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#validations